### DISCUSSION TITLE
Question on git remote access

### DISCUSSION THREAD

**Discussion by miinecrafter** [community/community#136415] [New to GitHub]
So I have this remote for my own personal repo on a computer I no longer use.

Would "git remote rm origin" prevent the user of the computer from committing to my github account? If not then could anyone direct me to how to prevent that device from committing?

Also, I would prefer a method which could do this without interacting with the device in anyway, maybe through my github account itself, but if that is impossible I can use the original device.

Thanks

**Comment by ygor-sena** [comment]
Hello @miinecrafter! The command you provided would not work because it doesn't affect the remote repository or the permissions of the other computer.

I think you can try to:

1. _Remove the SSH keys that are associated with the old device_, if you used one. To do this, go to your profile **Settings > SSH and GPG keys > Delete** the associated key
2. _Change your GitHub password_, which will invalidate any saved credentials that the old device might be using. To do this, go to your profile **Settings > Password and authentication > Change Password**
